sp800-63b/sec5_authenticators.md

```diff
@@ -356,7 +356,7 @@ The biometric system SHALL allow no more than 3 consecutive failed authenticatio
 
 **At AAL2**, the verifier SHOULD make a determination that the biometric sensor and subsequent processing meet the performance requirements stated above. **At AAL3**, this determination SHALL be performed.
 
-For any AAL, determination of sensor/endpoint performance, integrity, and authenticity can be accomplished in several different ways, any of which are acceptable under this guideline.  These include but are not limited to: authentication of the sensor or endpoint, testing and certification, and runtime interrogation of signed metadata (attestation) as described in Section [5.2.4](#attestation).
+For any AAL, determination of sensor/endpoint performance, integrity, and authenticity can be accomplished in several different ways, any of which are acceptable under this guideline.  These include but are not limited to: authentication of the sensor or endpoint, testing and certification, or runtime interrogation of signed metadata (attestation) as described in Section [5.2.4](#attestation).
```

Comment:
Not a huge deal, but I had used 'and' here because all three of those methods are examples of acceptable ways to do this.